iml-manager-cli/src/server.rs

```diff
@@ -95,6 +101,15 @@ fn filter_known_hosts<'a, 'b>(
         .collect()
 }
 
+fn can_continue(config: &AddHosts, err_msg: &str) -> bool {
+    display_error(&err_msg);
+    config.force || (config.prompt && get_confirm())
+}
+
+fn get_confirm() -> bool {
+    clt::confirm("Continue deployment?", true, "> ", true)
```

Comment:
Maybe it's not worth a separate function.

Reply:
This function expected to be long but much thanks to clt crate :) I like separate variant because it makes can_continue() to have pure logic only.